pkgs/by-name/ag/age-plugin-xwing/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "age-plugin-xwing";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "Rixxc";
+    repo = pname;
+    rev = "v${version}";
```

```suggestion
    tag = "v${version}";
```

We can use `tag` now.